### PR TITLE
New package: MevLyshkin.uec version 0.2.0

### LINKED ISSUE #190139: [Package Request]: UEC- Unreal Engine CLI helper tool
### How can we help?

I would like help so I can submit the manifest.

### Please read and ensure the following

- [x] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/Leinnan/uec
Publisher: MevLyshin
Package Name: uec
Description: Unreal Engine CLI helper tool
Package Version: 0.2.0
Installer Url: https://github.com/Leinnan/uec/releases/download/0.2.0/uec-0.2.0-x86_64.msi

### PR DESCRIPTION
**Pull request by Leinnan**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #190139

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

Link to the issue: https://github.com/microsoft/winget-pkgs/issues/190139

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/190138)

### CODE REVIEW

**Comment by Leinnan** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Program Files\uec\bin\uec.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 897.)

**Comment by Leinnan** [comment]
@stephengillie Okay, what should I do with it?

File: manifests/m/MevLyshkin/uec/0.2.0/MevLyshkin.uec.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: MevLyshkin.uec
+PackageVersion: 0.2.0
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{089471CD-ED1E-48F0-828C-1E6C1E1DD035}'
+Installers:
```
Comment (stephengillie) [inline manifests/m/MevLyshkin/uec/0.2.0/MevLyshkin.uec.installer.yaml:9]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Manual Validation ended with:
image

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
Verify:
image

**Pull request merged**